### DISCUSSION TITLE
Postprocessing and MeshPhysicalMaterial transmission rendering issue in HalfFloat type

### DISCUSSION THREAD

**Discussion by epouliquen** [pmndrs/postprocessing#430] [Q&A]
Hi,

  it seems any MeshPhysicalMaterial using transmission has rendering issue when HDR image is seen in refraction : It creates "burning" artifacts. 

This appears only when the effectcomposer type is in HalfFloat or Float types. But using none of these types, refraction is ok but everything else has banding.

 I don't understand why this happens because reading Three js Renderers sources show that they use Halfloat type texture for refraction renderTarget...

 The issue happens for any kinf of effect pass of the postprocessing lib (except RenderPass) (I've tried with saturation, SSAO, ToneMapping etc).

Please try this codesandbox to reproduce burning artifacts : https://codesandbox.io/s/halffloat-transmission-issue-2j3hr7?file=/src/index.js

![Capture](https://user-images.githubusercontent.com/50987570/203627165-57ba6576-244b-44b7-ab35-1f41db78f006.JPG)

**Comment by vanruesc** [comment]
Continued in https://github.com/pmndrs/postprocessing/issues/431
